frontend_app_logbook/src/component/LogbookEntry.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+import {
+  formatAbsoluteDate,
+  Icon,
+  IconButton
+} from 'tracim_frontend_lib'
+
+const LogbookEntry = (props) => {
+  return (
+    <div
+      className='logbook__timeline__entries__entry'
+      style={{ backgroundColor: props.entry.bgColor }}
+    >
+      <div className='logbook__timeline__entries__entry__dot' />
+      <div
+        className='logbook__timeline__entries__entry__arrow'
+        style={{ borderRight: `0.5rem solid ${props.entry.bgColor}` }}
```

Comment:
don't set static style in style html attribute

set the border-width and border-style in stylus file and only set the border-color with the html attribute

also, avoid rem, prefer px

Reply:
fixed